.rubocop.yml

```diff
@@ -220,9 +220,19 @@ Rails/HasAndBelongsToMany:
 Rails/WhereExists:
   Enabled: false
 
-# (0.87) pending cop
+# Enable with default EnforcedStyle: grouped; saves lines in some classes
 Style/AccessorGrouping:
```

Comment:
@pellaea @nimmolo @mo-nathan:
In enabling the cop, I decided to stick with the default configuration:
```ruby
EnforcedStyle: grouped (default)
# bad

class Foo
  attr_reader :bar
  attr_reader :baz
end

# good
class Foo
  attr_reader :bar, :baz
end
```
This makes the classes (marginally) shorter, and because it's presumably more widely accepted.
I don't have strong feelings about this.  If you'd rather use the [opposite configuration](https://docs.rubocop.org/rubocop/1.29/cops_style.html#enforcedstyle-separated) please let me know.
Another alternative is disabling the cop entirely.